Controllers/UserController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace HK2TProject_HotelManage_Server.Controllers
+{
+   [Route("api/[controller]")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+        private readonly IConfiguration _configuration;
+        private readonly ApplicationDbContext _context;
+
+        public AuthController(IConfiguration configuration, ApplicationDbContext context)
+        {
+            _configuration = configuration;
+            _context = context;
+        }
+
+        [HttpPost("login")]
```

Comment:
sẽ có đến 2 api login , 1 là login cho customer riêng và cho nhân viên (rule khác customer) riêng , chỉnh lại endpoint thành /sign-in/admin và cover lại code